Scripts/ControlsManager.js

```diff
@@ -0,0 +1,89 @@
+"use strict";
+
+//requires d3.js 
+
+export default function ControlsManager(){
+  const starting_theta = 45;
+  const starting_phi = 0;
+
+  let
```

Comment:
What is the purpose of "let" here?

Reply:
Here it does the same thing as "var", because in this case block scope and function scope are the same. This is my bad because I should have been consistent with this (in the other two classes they are "var").